sessionctx/variable/tidb_vars.go

```diff
@@ -829,6 +829,8 @@ const (
 	// TiDBEnablePlanCacheForSubquery controls whether prepare statement with subquery can be cached
 	TiDBEnablePlanCacheForSubquery = "tidb_enable_plan_cache_for_subquery"
 
+	// TiDBEnableLateMaterialization indicates whether to enable late materialization
+	TiDBEnableLateMaterialization = "tidb_enable_late_materialization"
```

Comment:
Just for easy to collect the vars for optimizer, the var is better to begin with `tidb_opt`.

Reply:
revised